src/components/SidebarSectionList.js

```diff
@@ -0,0 +1,48 @@
+import React from "react"
+import classnames from "classnames"
+
+function Link({ icon, text, href, className }) {
```

Comment:
I think that the Link component should be merged with the Item component. Both has the same class names, the only thing that changes is that in one case there is an `a` link around.

We can either always use Item, and display the `a` if there is an `href` prop or enforce the use of the Link, but in this case we should mutualize the classes used for the `li` icon and text elements

Reply:
that was my first approach, but i found it more messy, so splitting to link and item felt more cleaner to me, it's not that problem to merge them if you prefer it that way

Reply:
In that case if you prefer having two components, we should at least extract the className that are duplicated to a var :) Something like:

```
const liClassName = "flex flex-row mt-3 font-medium text-sm text-gray-800"

// ...

<li className={liClassName}>
```